src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.cs

```diff
@@ -14,6 +14,9 @@ namespace Windows.UI.Xaml.Controls
 	public partial class DatePicker : Control
 	{
 		public event EventHandler<DatePickerValueChangedEventArgs> DateChanged;
+		public event EventHandler<DatePickerSelectedValueChangedEventArgs> SelectedDateChanged;
+
+		private static readonly DateTimeOffset UnsetDateValue = DateTimeOffset.MinValue;
```

Comment:
Shouldn't this be set to the date of the day instead of the min one ? When  do you want the date to be at its minimum ?

Reply:
So the explaination is right above the DateProperty DependencyProperty but I feel like it's more a workaround than the desired behavior.
Also, what is #18331 ? Can't find it on Github or Azure DevOps.

Reply:
>https://dev.azure.com/nventive/Umbrella/_workitems/edit/30920
I leave this as is to avoid creating further breaking changes. 
CC @davidjohnoliver

Reply:
The default selection day has been changed to "Today" when unspecified. This is handled here:
```
_flyout.Opened += (s, e) => _flyout.Date = SelectedDate ?? DateTimeOffset.Now;
```
Since we now have `Date` and `SelectedDate`, with the later being a nullable version of the other. The `UnsetDateValue` is just a default value to coalesce/reset `Date` when `SelectedDate` is to `defualt(DateTimeOffset?)`,

For uwp, this value is: `new DateTimeOffset(new DateTime(1601, 1, 1), TimeSpan.Zero).ToLocalTime()`.